docs/content/contributing/HowToStart.md

```diff
@@ -0,0 +1,19 @@
+# How to start
+
+## Project structure
+
+- `subprojects` contains almost all of source code divided by project modules
+
+- `samples` separate gradle project that includeBuild("subprojects") to iterate on changes fast and provide samples
+
+- `ci/` and `/*.sh` - we follow [IaC](https://en.wikipedia.org/wiki/Infrastructure_as_code) principle whenever possible.
+  You can see docker images we use to abstract configuration of apps building and testing, as well as testing github
+  project itself.
+- `docs/` - documentation you see right now and code to deploy it automatically.  
+  [How we document](contributing/Documentation.md)
+
```

Comment:
We could also mention here to take a look into the makefile. It contains many helpful shortcuts.